web_tree_many2one_clickable/static/src/js/web_tree_many2one_clickable.js

```diff
@@ -45,11 +39,10 @@ ListView.Column.include({
         if (this.widget == 'many2one_clickable') {
             this.use_many2one_clickable = true;
         } else if (this.type == 'many2one') {
-            if (use_many2one_clickable_default) {
-                this.use_many2one_clickable = true;
-            } else {
-                get_options();
-            }
+            this.use_many2one_clickable = false;
+            clickable_default($.proxy(function(value){
+                this.use_many2one_clickable = value;
+            }, this));
```

Comment:
OK, I get your point of just asking once and only when the widget gets initialized, and it's nice, but let me try to do this more straightforward.

This would go above:

```
var asked = false, default = $.Deferred();

function ask_once() {
  if (asked) {return default}
  asked = true;
  new Model("ir.config_parameter")
            .call("get_param", ["web_tree_many2one_clickable.default", false])
            .done(function(value){
                default.resolve(String(value).toLowerCase() === "true");
            })
            .fail(function(){
                default.reject();
             });
}
```

Then, here, inside the `init()`:

```
this.use_many2one_clickable = false;
ask_once();
default.done($.proxy(function(value){
                this.use_many2one_clickable = value;
            }, this));
```

It makes the same, but all is easier to understand.